docs/_static/copy.js

```diff
@@ -0,0 +1,22 @@
+const COPY = "Copy";
+const COPIED = "Copied!";
+
+const copy = async (obj) => {
+  await navigator.clipboard.writeText(obj.children[1].innerText);
+
+  obj.children[0].textContent = COPIED;
+  setTimeout(() => (obj.children[0].textContent = COPY), 2500);
+};
+
+document.addEventListener("DOMContentLoaded", () => {
+  let allCodeblocks = document.querySelectorAll("div[class='highlight']");
+
+  for (let codeblock of allCodeblocks) {
+      codeblock.parentNode.className += " relative-copy";
+      let copyEl = document.createElement("span");
+      copyEl.onclick = () => copy(codeblock);
```

Comment:
Nothing wrong with this, but it's more customary to use `addEventListener("click", ...)`.